VAMobile/e2e/tests/ContactInformation.e2e.ts

```diff
@@ -18,47 +18,79 @@ export const ContactInfoE2eIdConstants = {
   STATE_ID: 'stateTestID',
   ZIP_CODE_ID: 'zipCodeTestID',
   PHONE_NUMBER_EXTENSION_ID: 'phoneNumberExtensionTestID',
-  PHONE_NUMBER_ID: 'phoneNumberTestID'
+  PHONE_NUMBER_ID: 'phoneNumberTestID',
+  CANCEL_DELETE_TEXT: device.getPlatform() === 'ios' ? 'Delete Changes' : 'Delete Changes ',
+  CANCEL_KEEP_EDITING_TEXT: device.getPlatform() === 'ios' ? 'Keep Editing' : 'Keep Editing ',
+  REMOVE_KEEP_TEXT: 'Keep',
+  REMOVE_REMOVE_TEXT: 'Remove',
+  SAVE_TEXT: 'Save',
+  DISMISS_TEXT: 'Dismiss'
 }
 
-beforeAll(async () => {
-  await loginToDemoMode()
-  await openProfile()
-  await openContactInfo()
-})
+export async function updateAddress() {
+  await element(by.id(ContactInfoE2eIdConstants.STREET_ADDRESS_LINE_2_ID)).tapReturnKey()
+  await element(by.id(ContactInfoE2eIdConstants.COUNTRY_PICKER_ID)).tap()
+  await element(by.text('United States')).tap()
+  await element(by.text('Done')).tap()
+  await element(by.id(ContactInfoE2eIdConstants.STREET_ADDRESS_LINE_1_ID)).typeText('3101 N Fort Valley Rd')
+  await element(by.id(ContactInfoE2eIdConstants.STREET_ADDRESS_LINE_1_ID)).tapReturnKey()
+  await element(by.id('EditAddressTestID')).scrollTo('bottom')
+  await element(by.id(ContactInfoE2eIdConstants.CITY_TEST_ID)).replaceText('Flagstaff')
+  await element(by.id('stateTestID')).tap()
+  await element(by.text('Arizona')).atIndex(0).tap()
+  await element(by.text('Done')).tap()
+  await element(by.id(ContactInfoE2eIdConstants.ZIP_CODE_ID)).replaceText('86001')
+}
 
-describe('Contact Info Screen', () => { 	
-  it('should match the Contact Info page design', async () => {
-		await expect(element(by.id(ContactInfoE2eIdConstants.MAILING_ADDRESS_ID))).toExist()
-		await expect(element(by.id(ContactInfoE2eIdConstants.HOME_ADDRESS_ID))).toExist()
-    await expect(element(by.id(ContactInfoE2eIdConstants.HOME_PHONE_ID))).toExist()
-		await expect(element(by.id(ContactInfoE2eIdConstants.WORK_PHONE_ID))).toExist()
-    await expect(element(by.id(ContactInfoE2eIdConstants.MOBILE_PHONE_ID))).toExist()
-		await expect(element(by.id(ContactInfoE2eIdConstants.EMAIL_ADDRESS_ID))).toExist()
-		await expect(element(by.text(ContactInfoE2eIdConstants.HOW_WE_USE_TEXT))).toExist()
-	}) 
+export async function validateAddresses(addressID, addressType) {

```

Comment:
Mailing address is passing, but all of the home address cases are failing (iOS) starting here

Reply:
yep, resolved.